server/models/Event.ts

```diff
@@ -0,0 +1,46 @@
+import { model, models, Model, Schema, Document } from "mongoose";
+import ContentfulImageSchema from "./ContentfulImageSchema";
+import { Event } from "utils/types";
+
+export const EventSchema = new Schema({
+    name: {
+        type: String,
+        required: true,
+    },
+    description: {
+        type: String,
+        required: true,
+    },
+    start_time: {
+        type: Date,
+        required: true,
+    },
+    end_time: {
+        type: Date,
+        required: true,
+    },
+    start_registration: {
+        type: Date,
+        required: false,
+    },
+    end_registration: {
+        type: Date,
+        required: false,
+    },
+    location: {
+        type: String,
+        required: true,
+    },
+    hours: {
+        type: String,
+        required: true,
+    },
+    image: {
+        type: ContentfulImageSchema,
+        required: false,
+    },
+});
```

Comment:
Do we need to add the signed up attendees, present attendees, and absent attendees properties?

Reply:
We do. I haven't finalized how we should store them yet (maybe _ids of attendees, maybe denormalized to only hold the info we need from the volunteers type). Definitely will need to get updated when we figure it out though,